### PR TITLE
Style

### PR DESCRIPTION
**Pull request by hedita**

### CODE REVIEW

File: src/components/MusicItem/MusicItem.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import list from "../../assets/list.json";
+import "./MusicItem.scss";
+
+const MusicItem = () => {
+  const musicList = list.albums.items;
+
+  const musics = musicList.map(({ name, id, images }) => {
```
Comment (blacksrc) [inline src/components/MusicItem/MusicItem.jsx]: You can skip defining variable here.

File: src/components/MusicItem/MusicItem.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import list from "../../assets/list.json";
+import "./MusicItem.scss";
+
+const MusicItem = () => {
+  const musicList = list.albums.items;
```
Comment (blacksrc) [inline src/components/MusicItem/MusicItem.jsx]: This variable seems unnecessary.

File: src/components/PlayListItem/PlayListItem.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import playlists from "../../assets/Playlists.json";
+import "./PlayListItem.scss";
+
+const PlayListItem = () => {
+  const playListsItems = playlists.playlists.items;
```
Comment (blacksrc) [inline src/components/PlayListItem/PlayListItem.jsx]: This variable seems unnecessary.

File: src/components/PlayListItem/PlayListItem.jsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+  const playListItem = playListsItems.map(({ external_urls, id, name }) => {
```
Comment (blacksrc) [inline src/components/PlayListItem/PlayListItem.jsx]: This variable seems unnecessary.

File: src/components/SinglesItem/SinglesItem.jsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import list from "../../assets/list.json";
+
+const SinglesItem = () => {
+  const singlesList = list.albums.items;
```
Comment (blacksrc) [inline src/components/SinglesItem/SinglesItem.jsx]: This variable seems unnecessary.

File: src/components/SinglesItem/SinglesItem.jsx [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+  const result = singlesList.map(({ name, album_type, id, images }) => {
```
Comment (blacksrc) [inline src/components/SinglesItem/SinglesItem.jsx]: This variable seems unnecessary.

Comment (hedita) [reply src/components/SinglesItem/SinglesItem.jsx]: Done.

File: src/components/SinglesItem/SinglesItem.jsx (hunk 5 above)
Comment (hedita) [reply src/components/SinglesItem/SinglesItem.jsx]: Done.

File: src/components/PlayListItem/PlayListItem.jsx (hunk 4 above)
Comment (hedita) [reply src/components/PlayListItem/PlayListItem.jsx]: Done.

File: src/components/PlayListItem/PlayListItem.jsx (hunk 3 above)
Comment (hedita) [reply src/components/PlayListItem/PlayListItem.jsx]: Done.

File: src/components/MusicItem/MusicItem.jsx (hunk 2 above)
Comment (hedita) [reply src/components/MusicItem/MusicItem.jsx]: Done.

File: src/components/MusicItem/MusicItem.jsx (hunk 1 above)
Comment (hedita) [reply src/components/MusicItem/MusicItem.jsx]: Done.

File: src/components/PlayListItem/PlayListItem.jsx [hunk 7]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import playlists from "../../assets/Playlists.json";
+import "./PlayListItem.scss";
+
+const PlayListItem = () => {
+  return playlists.playlists.items.map(({ external_urls, id, name }) => {
```
Comment (takrami) [inline src/components/PlayListItem/PlayListItem.jsx]: move the map inside the parent component. just u did for other components.

File: src/components/MusicItem/MusicItem.scss [hunk 8]
```diff
@@ -0,0 +1,6 @@
+@import "../../components/Variables.scss";
+
+.music-list {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
```
Comment (takrami) [inline src/components/MusicItem/MusicItem.scss]: the list needs to be responsive. use grid in a way that u have responsive web page.

File: src/components/SinglesList/SinglesList.scss [hunk 9]
```diff
@@ -0,0 +1,10 @@
+@import "../../components/Variables.scss";
+
+.singles-list {
+  margin-left: $sizing-s;
+}
+
+.music-list {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+}
```
Comment (takrami) [inline src/components/SinglesList/SinglesList.scss]: this is dublicated.

File: src/pages/Albums&Singles/AlbumsSingles.jsx [hunk 10]
```diff
@@ -1,15 +1,15 @@
-import React from "react";
+// import React from "react";
```
Comment (takrami) [inline src/pages/Albums&Singles/AlbumsSingles.jsx]: if u don't need the file please remove it.

File: src/components/PlayListItem/PlayListItem.jsx (hunk 7 above)
Comment (hedita) [reply src/components/PlayListItem/PlayListItem.jsx]: Done.

File: src/components/MusicItem/MusicItem.scss (hunk 8 above)
Comment (hedita) [reply src/components/MusicItem/MusicItem.scss]: Done.

File: src/components/SinglesList/SinglesList.scss (hunk 9 above)
Comment (hedita) [reply src/components/SinglesList/SinglesList.scss]: Oops!

File: src/pages/Albums&Singles/AlbumsSingles.jsx (hunk 10 above)
Comment (hedita) [reply src/pages/Albums&Singles/AlbumsSingles.jsx]: Done.

**Pull request merged**